README.md

```diff
@@ -1,24 +1,86 @@
-# README
+# Enumerable-Methods
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This project illustrates multiple ways to build a form in rails that can send parameters to a controller so that it can build an instance of a model with those parameters.
 
-Things you may want to cover:
+![screenshot](https://raw.githubusercontent.com/JuanDavid31/re-former/first_forms/app_screenshot.PNG)
 
-* Ruby version
+## Built With
 
-* System dependencies
+- Ruby on Rails
+- Using Linters for Ruby and Stylelint
+- Git, GitHub
+- Using Bundler
 
-* Configuration
+## Getting Started
 
-* Database creation
+To get a local copy up and running follow these simple steps.
 
-* Database initialization
+- Go to the main page of te repo. "git clone https://github.com/JuanDavid31/re-former.git"
+- Press the "Code" button and get the repo link.
+- Clone it using git.
+- Open terminal on that folder.
+- run `yarn install` 
+- run `npm install`
+- run `rails webpacker:install`
+- run `bundle install`
+- run `rails db: migrate` to migrate the database 
+- run `rails server`
+- Go to `http://localhost:3000` on your browser to view the homepage of the application 
```

Comment:
- Add instruction for accessing the sign up page. `localhost:3000/users/new`
- Add the path for editing user. `localhost:3000/users/{id}/edit`